pkg/services/rpcsrv/client_test.go

```diff
@@ -2466,6 +2466,26 @@ func TestClient_GetVersion_Hardforks(t *testing.T) {
 	require.InDeltaMapValues(t, expected, v.Protocol.Hardforks, 0)
 }
 
+func TestClient_GetVersion_AllFields(t *testing.T) {
```

Comment:
This test doesn't test all fields. So either rename or ensure it tests all fields.